microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -645,6 +645,18 @@ By default, while browsing **InPrivate**, Microsoft does not collect any informa
 Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.  You can stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing. Open **Start** > **Settings** > **Privacy** and turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).
 
 
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When working with the work profile, the organization can see the following data:
+* The application URL.
+* The identity of the user in the profile.
+* The file name.
```

Comment:
@apryzgoda - need more context for "the file name".  Which file name, of what?

Reply:
'The content of files': The content of files the user attempts to download or upload.

Reply:
@AmitMishaeli - regarding specifically the list item "The file name", I think clarifying for context might give:

The file name of files that the user attempts to download or upload.

correct?  is there better wording?